src/components/album/Album.css

```diff
@@ -0,0 +1,24 @@
+.album{
+  color: white;
+  padding: 10px;
+  justify-content: center;
+  align-items: center;
+  height: 360px;
+  margin: 0 0 0 0;
```

Comment:
When all of the margins are the same, you could just write one number. `margin: 0` is the same as `margin: 0 0 0 0`